### PR TITLE
I tried my best

### PR DESCRIPTION
**Pull request by dominikkawka**
@laurafitzgerald

### CODE REVIEW

**Review by laurafitzgerald — CHANGES_REQUESTED** [review]
changes look fine @dominikkawka branch is out of date however. can you rebase with master and update this pr.

File: README.md [hunk 1]
```diff
@@ -10,8 +10,12 @@ Add your name and and one other line here:
 
 Jameel Briones
 - I love cats!
+
```
Comment (laurafitzgerald) [inline README.md:13]: can you remove this extra line

**Review by laurafitzgerald — CHANGES_REQUESTED** [review]
looks good, just one comment to fix

File: README.md [hunk 2]
```diff
@@ -10,9 +10,12 @@ Add your name and and one other line here:
...
 David Kirwan
 Hi I'm currently bashing my head off OSx upgrades for OSx and XCode
 
-Dominik Kawka
-I like lasagna.
+Maciej Marchel
```
Comment (laurafitzgerald) [inline README.md]: Did this change get added manually or was it included through a rebase?

Comment (dominikkawka) [reply README.md]: It was through a rebase, I'll do it again

Comment (laurafitzgerald) [reply README.md]: It's fine if it was added through the rebase. could be just github picking up the chnage funny because the change is similar.

**Pull request closed without merging**